roles/KubevirtCommonTemplatesBundle/templates/kubevirt-base-image-editrole.yaml.j2

```diff
@@ -0,0 +1,17 @@
+apiVersion: rbac.authorization.k8s.io/v1
```

Comment:
The files in this directory should be renamed from `*-base-image-*` to `*-os-images-*`